docs/unified-test-documentation/generic-testing-stand-setup.md

```diff
@@ -0,0 +1,351 @@
+second black wire from the improper cable and solder it to the unstriped
+wire of the proper cable. Use tape to isolate and strengthen the whole
+connection. The cable is ready. Be sure to use plugs in the following way:
+
+![](../images/cable_connectors.jpg)
+
+### Sonoff setup
+
+If you require line voltage control, follow our guide for [Sonoff preparation](https://docs.dasharo.com/transparent-validation/sonoff/sonoff_preparation/).
+
+### PiKVM setup
+
+If serial connection to the DUT is known to be problematic, follow our guide
+for [PiKVM preparation](https://docs.dasharo.com/transparent-validation/pikvm/assembly-and-validation/).
+
+### Access to the DUT
+
+Access to the DUT should be realized by connecting the serial port on the DUT
+to the serial port on RTE. The location of the serial port should be determined
+based on the platform's documentation. Documentation describing this process and
+including connections with various cables can be found
+[here](https://docs.dasharo.com/transparent-validation/rte/v1.1.0/serial-port-connection-guide/).
+
+Follow the steps below to configure `ser2net` on RTE, which will allow you to
+access the DUT via serial using the telent console. In this example scenario, a
+micro USB-USB converter is used to connect the DUT with RTE.
+
+1. Connect to RTE via ssh.
+1. Run the `dmesg -w` command.
+1. Disconnect and connect a micro USB-USB converter.
+1. On the `dmesg -w` command output look for the attached converter, in this
+   case:
+
+    ```sh
+    [  164.136255] usb 8-1: f81232 converter now attached to ttyUSB0
+    ```
+
+1. Then use vim to modify settings in `/etc/ser2net.conf` according to the port
+   received from the `dmesg -w` command, in this case(`ttyUSB0`):
+
+    ```sh
+    13541:telnet:1200:/dev/ttyUSB0:115200 8DATABITS NONE 1STOPBIT
+    ```
+
+    ```sh
+    13542:telnet:1200:/dev/debug_uart_converter:115200 8DATABITS NONE 1STOPBIT
+    ```
+
+1. Check access to the DUT using the `telnet <IP> <port>` command.
+
+In case it is not possible to read the device via serial, set up PiKVM and
+properly connect to the platform. PiKVM setup documentation can be found
+[here](https://docs.dasharo.com/transparent-validation/pikvm/assembly-and-validation/).
+
+#### Platform external flashing
+
+Platform external flashing is needed for two reasons:
+
+* it enables quick changes to the firmware,
+* it enables the process of unbricking the platform.
+
+The flashing operation usually consists of several commands, involving crucial
+power management, so it is advisable to prepare a script that automates the
+process. This is a crucial function, so if in doubt, ask your supervisor.
+
+> NOTE: On some platforms, the flash chip supply voltage is 1.8V, not 3.3V.
+  The supply voltage should be determined before making any connections, based
+  on our platform-specific documentation or the flash chip datasheet.
+
+Example script template:
+
+```bash
```

Comment:
I am not a huge fan of distributing this script.
Can we at least recommend to use https://github.com/Dasharo/osfv-scripts/blob/main/osfv_cli/README.md first? we can improve the documentation or the tool if needed. All of the commands available in the tool are documented in help. Most Protectli platforms should be supported.

Reply:
Example

```
macpijan in ~/projects/github/dasharo/osfv-scripts/osfv_cli on main ● λ osfv_cli rte --rte_ip 192.168.10.244 flash probe
DUT model retrieved from snipeit: VP4630
Probing flash...
Executing command: flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -c MX25L12835F/MX25L12845E/MX25L12865E
flashrom v1.2 on Linux 5.4.69 (armv7l)
flashrom is free software, get the source code at https://flashrom.org


Using clock_gettime for delay loops (clk_id: 1, resolution: 1ns).
Found Macronix flash chip "MX25L12835F/MX25L12845E/MX25L12865E" (16384 kB, SPI) on linux_spi.

No operations were specified.


macpijan in ~/projects/github/dasharo/osfv-scripts/osfv_cli on main ● λ osfv_cli rte --rte_ip 192.168.10.244 flash read --rom vp4630-read.rom
DUT model retrieved from snipeit: VP4630
Reading from flash...
Executing command: flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -c MX25L12835F/MX25L12845E/MX25L12865E -r /tmp/read.rom
flashrom v1.2 on Linux 5.4.69 (armv7l)
flashrom is free software, get the source code at https://flashrom.org

Using clock_gettime for delay loops (clk_id: 1, resolution: 1ns).

Found Macronix flash chip "MX25L12835F/MX25L12845E/MX25L12865E" (16384 kB, SPI) on linux_spi.

Reading flash... 
done.


Read flash content saved to vp4630-read.rom
```

Reply:
[Replaced](https://github.com/Dasharo/docs/pull/753/commits/3b1e9b8738ab810b772a98c7100876b148565496#diff-21b728890d92d73a2d9a1746109ea2d9309b6b02efbcf910fbe8ffc652a6c269R228) by link to osfv-cli and example use case

Reply:
Thanks @filipleple. 
This should be, hoverer, generalized. This lambda prompt is my prompt, is not a common one. We should use .e.g `$`. We should also not provide exact IP address, but a placeholder.